src/librbd/Utils.cc

```diff
@@ -89,7 +91,7 @@ uint64_t parse_rbd_default_features(CephContext* cct)
         value += conf_vals[feature];
       } else {
         ret = -EINVAL;
-        ldout(cct, 1) << "Warning: unknown rbd feature " << feature << dendl;
+        lderr(cct) << "unknown rbd feature " << feature << dendl;
```

Comment:
Note that this falls back to the defaults on -EINVAL below. That was the rationale for the warning since execution will continue.